tjpcov/covariance_cluster_counts.py

```diff
@@ -0,0 +1,117 @@
+from .covariance_clusters import CovarianceClusters
+import numpy as np
+import pyccl as ccl
+from scipy.integrate import romb
+
+
+class ClusterCounts(CovarianceClusters):
```

Comment:
Why is this separate of the `CovarianceClusters`? Is `CovarianceClusters` general enough for other cases?

Reply:
That's the idea - move the cluster covariance into a base class, and cluster counts just implements that.  In future PR for N x C_ell I can shift stuff around. Or if it doesnt make sense to have an abstract class then we can just remove it.

Reply:
Sounds good